### PR TITLE
Components

### PR DESCRIPTION
**Pull request by sntgopl**
# Review request

### In this milestone I have:

- Created a component 'Calculator' to render all buttons on the screen.
- Added CSS style to the board according to the UI given by the template.
- Import React to render in the index.js

### CODE REVIEW

**Review by Reem-lab — COMMENTED** [review]
Hi @sntgopl, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have linters configuration so kindly add the linters files into your project and set them all, And kindly customize your README with your project information😊🙏🏻

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by FrederickMih — COMMENTED** [review]
Hi @sntgopl , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive README.MD file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="application">
+      <Calculator />
     </div>
   );
 }
```
Comment (Bondok6) [inline src/App.js]: - [ ] Great job here 💯 But according to the requirements, you should use class based components (All components should be class-based). so Kindly make the `App.js` component  `class-based`. 👍

File: README.md [hunk 2]
```diff
@@ -1,70 +1,61 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](./LICENSE) licensed.
```
Comment (Bondok6) [inline README.md:61]: - [ ] The link to MIT/LICENSE file is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @sntgopl 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Your README is professional ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @sntgopl  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes as requested by the previous reviewer 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**